common/libs/nodes/rpc/client.ts

```diff
@@ -22,22 +22,30 @@ export default class RPCClient {
   public call = (request: RPCRequest | any): Promise<JsonRpcResponse> => {
     return fetch(this.endpoint, {
       method: 'POST',
-      headers: {
+      headers: this.createHeaders({
         'Content-Type': 'application/json',
-        ...this.headers,
-      },
+        ...this.headers
+      }),
       body: JSON.stringify(this.decorateRequest(request))
     }).then(r => r.json());
   };
 
   public batch = (requests: RPCRequest[] | any): Promise<JsonRpcResponse[]> => {
     return fetch(this.endpoint, {
       method: 'POST',
-      headers: {
+      headers: this.createHeaders({
         'Content-Type': 'application/json',
-        ...this.headers,
-      },
+        ...this.headers
+      }),
       body: JSON.stringify(requests.map(this.decorateRequest))
     }).then(r => r.json());
   };
+
+  private createHeaders = headerObject => {
```

Comment:
Should type up `headerObject` with the appropriate header keys

Reply:
@HenryNguyen5 The headerObject been console.logging as blank, aside from the content-type, but previously the code was written to loop around the Object keys, so I wrote it this way for compatibility reasons.